t/mojolicious/lib/TestApp.pm

```diff
@@ -0,0 +1,10 @@
+package TestApp;
```

Comment:
This class doesn't really add much value, does it? Couldn't you just as well have used `Mojolicious::Lite` in the test file?